tests/correctness/numeric-types/fixed-point/diff-width-add.futil.data

```diff
@@ -0,0 +1,44 @@
+{
+  "mem0": {
+    "data": [
+        [1.5,0,0,0],
```

Comment:
Same question as above—these zeros should probably be actual values. Good opportunity to document overflow behavior.

Reply:
Agreed, I copied this over from Yoona's PR. I'm going to remove it for now. I want to test the primitives in a more systematic way, i.e. For each numeric type, make a list of corner cases and our expectations. I plan to write this up.